### ISSUE TITLE
running xbow-cluser in fresh setup fails with image id

### ISSUE THREAD

**Issue by simonbennie** [ChrisSuess/Project-Xbow#17]
Having followed the install guide I'm getting a crash when running xbow-cluster

```
xbow-create_cluster
Starting the scheduler node - this may take some time...
Traceback (most recent call last):
  File "/Users/simonbennie/gits/cocomd/venv/bin/xbow-create_cluster", line 36, in <module>
    user_data=user_data
  File "/Users/simonbennie/gits/cocomd/venv/lib/python3.6/site-packages/xbow/instances.py", line 354, in create
    ClientToken=str(uuid.uuid4()), MaxCount=1, MinCount=1)[0]
  File "/Users/simonbennie/gits/cocomd/venv/lib/python3.6/site-packages/boto3/resources/factory.py", line 520, in do_action
    response = action(self, *args, **kwargs)
  File "/Users/simonbennie/gits/cocomd/venv/lib/python3.6/site-packages/boto3/resources/action.py", line 83, in __call__
    response = getattr(parent.meta.client, operation_name)(**params)
  File "/Users/simonbennie/gits/cocomd/venv/lib/python3.6/site-packages/botocore/client.py", line 314, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/simonbennie/gits/cocomd/venv/lib/python3.6/site-packages/botocore/client.py", line 612, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (InvalidAMIID.NotFound) when calling the RunInstances operation: The image id '[ami-813a3a6b]' does not exist
```

This was run on osx with a fresh virtualenv setup with a correctly installed aws instance.

This was the settings.yml generated by xbow
```
region: eu-west-1
price: '0.05'
worker_instance_type: c5.large
scheduler_instance_type: t2.small
image_id: ami-813a3a6b # new image created 15/06/2018
ec2_security_groups: ['Xbow-SG-ec2']
efs_security_groups: ['Xbow-SG-mt']
shared_file_system: MyFileSystem
creation_token: MyFileSystem
mount_point: /home/ubuntu/shared
cluster_name: MyCluster
scheduler_name: MyScheduler
worker_pool_name: MyWorkers
pool_size: 1
```

**Comment by ChrisSuess** [comment]
Hi @simonbennie 

This appear to be a problem with our AMI being region specific.

Do you know which region you are running your instances from? In your home folder there should be a folder named `~/.aws/` in here there is a file named `config` which should tell you the region. Normally this defaults to a us-east-1 region which our custom AMI is not available in.

If you change the region in the file `~/.aws/config` to `eu-west-1` then this should work fine.

The file `~/.aws/config` should look as follows:
```
[default]
output = json
region = eu-west-1
```

Let me know if changing the region fixes the issue and I will amend the documentation.

Chris

**Comment by simonbennie** [comment]
Hi Chris,

My aws/config is already set to:

```
[default]
region = eu-west-1
```

**Comment by ChrisSuess** [comment]
Hi @simonbennie 

I can see a filesystem was created in the Frankfurt region (`eu-central-1`). When I try to boot Xbow in this region I get the same error message as our custom AMI is linked to the Ireland region (`eu-west-1`).

Is it possible you have a conflicting `~/.aws/config` file?

Chris

**Comment by simonbennie** [comment]
@ChrisSuess 

I looked through my history and I did set everything to eu-central-1, I'm not sure what is going on but I reran setup again with the same config in the existing venv and this time it worked. 

(venv) sh-3.2$ bin/xbow-configure
Settings.yml already exists at '~/.xbow, xbow is skipping creating a new one.
The security key SG-Xbow already exists
The security key SG-Xbow already exists
(venv) sh-3.2$ xbow-create_filesystem
Creating your filesystem
Checking targets for availability
Mount Targets are Available
FileSystem Created
(venv) sh-3.2$ xbow-create_cluster
Starting the scheduler node - this may take some time...
Scheduler ready, now starting workers. This may take some time...
Instance running, now waiting until xbow-cluster is accessible...

I'll try doing another clean install and see if i can reproduce
